bin/migrate-oats-data/applications/sql/application-populate/application_pop_fee.sql

```diff
@@ -0,0 +1,16 @@
+WITH
```

Comment:
Please provide some comments to SQL, for example, why only records with fee_amount equal to NUll are selected?